src/java/org/apache/cassandra/cache/AutoSavingCache.java

```diff
@@ -353,20 +353,15 @@ public void saveCache()
             {
 
                 //Need to be able to check schema version because CF names are ambiguous
-                UUID schemaVersion = Schema.instance.getVersion();
```

Comment:
The schemaVersion can no longer be null given the changes in this ticket, but how do we know it is not the EmptyVersion?  This code is protecting against the version not being setup yet, how is it ensured that it is set after these changes?

Reply:
I guess?  I am just going on the fact that there is explicit code here to "figure out the schema version" if it was null previously.  The way it would be null previously is that it was the "default value" of the attribute.  The new default value of the attribute is the EmptyVersion.  So indeed you don't need a null check anymore, but now should this be a check for EmptyVersion instead?